libturpial/lib/services/url/shortypython/imports.py

```diff
@@ -1,11 +1,7 @@
-## Shorty
-## Copyright 2009 Joshua Roesslein
-## See LICENSE
```

Comment:
Please, don't remove the copyright.

Reply:
it got removed by the 2to3 tool. I'll restore it